.github/workflows/ci-cd.yml

```diff
@@ -240,7 +241,7 @@ jobs:
 
     - name: Setup Python ${{ matrix.pyver }}
       id: python-install
-      uses: actions/setup-python@v5
+      uses: quansight-labs/setup-python@v5
```

Comment:
I think this can be reverted now, I saw a mention that they made a new release this week with support for the t version.